test/testSignVerify.js

```diff
@@ -0,0 +1,62 @@
+import * as EcdsaMultikey from '@digitalbazaar/ecdsa-multikey';
+import {createCryptosuite} from '../lib/create.js';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+import {ecdsaMultikeyKeyPair} from './mock-data.js';
+import jsigs from 'jsonld-signatures';
+import {loader} from './documentLoader.js';
+
+const {purposes: {AssertionProofPurpose}} = jsigs;
+
+// import the ECDSA key pair to use when signing
+const publicKeyMultibase = 'zDnaekGZTbQBerwcehBSXLqAg6s55hVEBms1zFy89VHXtJSa9';
+const controller = `did:key:${publicKeyMultibase}`;
+
+const keyPair = await EcdsaMultikey.from({...ecdsaMultikeyKeyPair});
+const documentLoader = loader.build();
+
+// create the unsigned credential
+const unsignedCredential = {
+  '@context': [
+    'https://www.w3.org/2018/credentials/v1',
+    {
+      '@protected': true,
+      AlumniCredential: 'urn:example:AlumniCredential',
```

Comment:
Align with the other definitions.
```suggestion
      AlumniCredential: 'https://example.org/examples#AlumniCredential',
```